### DISCUSSION TITLE
KIAUH multi-printer install: config path differences

### DISCUSSION THREAD

**Discussion by AndreyMarchuk** [bassamanator/Sovol-SV06-firmware#152] [Show and tell]
Did a fresh install on Orange Pi Zero 2W with 2 Klipper instances via KIAUH.
Named the printers `sovol06plus` and `3d` which created 2 data folders for each:
- `~/sovol06plus_data`
- `~/3d_data`

In this setup `printer.conf` is located under `~/sovol06plus_data/config`, not under `~/printer_data/config` used in the install instructions.
The easiest way to handle this is by creating a symlink for `~/sovol06plus_data/config`:
```
mv ~/sovol06plus_data/config ~/sovol06plus_data/config-original
cd ~/sovol06plus_data
ln -s ~/printer_data/config config
```

Hope this helps someone doing a similar setup.

**Comment by bassamanator** [comment]
This is useful, thanks!
